Description: detect ppc64el architecture correctly
Origin: qtwebkit, http://code.qt.io/cgit/qt/qtwebkit.git/commit?id=405022cf7bba9bb7
Forwarded: yes, https://codereview.qt-project.org/107469
Last-Update: 2015-03-01

--- a/src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h
+++ b/src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h
@@ -197,7 +197,9 @@
 #if   defined(__ppc64__) \
     || defined(__PPC64__)
 #define WTF_CPU_PPC64 1
+#ifndef __LITTLE_ENDIAN__
 #define WTF_CPU_BIG_ENDIAN 1
+#endif
 #endif
 
 /* CPU(SH4) - SuperH SH-4 */
